docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,79 @@
+@startuml
+'https://plantuml.com/sequence-diagram
```

Comment:
Once again I cannot thank you enough for helping the rest of the team by attaching the documentation website for the PUML sequence diagrams

Reply:
Do I sound like a business student enough?